ui/lib/pki/addon/components/pki-key-parameters.js

```diff
@@ -35,10 +36,9 @@ export default class PkiKeyParameters extends Component {
 
   @action onSignatureBitsOrKeyTypeChange(name, selection) {
```

Comment:
Keeping this function as is for now, but I'd like to convert this file to typescript and will address these methods accordingly!